packages/workflow/src/logs.ts

```diff
@@ -67,6 +75,7 @@ export const log: WorkflowLogger = Object.fromEntries(
         return loggerSink[level](message, {
           // Inject the call time in nanosecond resolution as expected by the worker logger.
           [LogTimestamp]: activator.getTimeOfDay(),
+          sdkComponent: SdkComponent.workflow,
```

Comment:
Do we need timeOfDayToBigint?

Reply:
Well observed… but we don't. The `getTimeDay()` function provided to the `activator`'s constructor already do `timeOfDayToBigint.